### DISCUSSION TITLE
求一个mgv2000的官方完整固件

### DISCUSSION THREAD

**Discussion by Aqr-K** [ophub/amlogic-s9xxx-armbian#1681] [Amlogic]
芯片：s905l3b 
网卡：7668 
内存：1G
EMMC：8G
广东移动，JL代工的
同cpu型号的也行，忘了备份，结果没有一个固件的dtb是完美兼容的，想试试自己官改个u-boot

**Comment by whelamc** [comment]
同款，你刷的哪个固件，我刷的任何一个固件都没法启动armbian

**Comment by ophub** [comment]
mt7668 wifi和蓝牙测试内核及使用方法： https://github.com/ophub/amlogic-s9xxx-armbian/issues/3610#issuecomment-5091652991
